lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/directories/DirectoriesException.java

```diff
@@ -0,0 +1,15 @@
+package org.enso.desktopenvironment.directories;
+
+import java.io.IOException;
+
+/** Indicates an issue when accessing user directories. */
+public final class DirectoriesException extends IOException {
```

Comment:
Any benefit of having own `IOException` subclass?

Reply:
Removed